packages/lodestar/src/network/reqresp/reqResp.ts

```diff
@@ -218,31 +217,43 @@ export class ReqResp implements IReqResp {
 
     switch (requestTyped.method) {
       case Method.Ping:
+        if (!this.responseRateLimiter.allowToProcess(peerId)) {
+          throw new ResponseError(RespStatus.SERVER_ERROR, "rate limit");
+        }
```

Comment:
Why not call once at the top of this function with 
```ts
       if (!this.responseRateLimiter.allowToProcess(protocol.method, peerId)) {
          throw new ResponseError(RespStatus.SERVER_ERROR, "rate limit");
        }
```

Reply:
Also, please review or ask Teku and Lighthouse if they use a different error code for rate limiting. I've seen before an unoficial list they mantain with different frequent error codes

Reply:
@dapplion Teku uses `INVALID_REQUEST`, we'll follow Lighthouse to use new `RATE_LIMITED = 139` error code

Reply:
regarding `allowToProcess()` call, `BeaconBlocksByRange` and `BeaconBlocksByRoot` put in different param (block count) compared to others